src/Achievements/AchievementCategory.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+
+import { Accordion, AccordionSummary, AccordionDetails, Typography } from "@mui/material";
+
+import { Achievement } from "./Achievements";
+
+interface IProps {
+  title: string;
+  achievements: { achievement: Achievement }[];
+  allAchievements?: { achievement: Achievement }[];
+  sx?: boolean;
+}
+
+function steamCount(achievements: { achievement: Achievement }[]): number {
+  return achievements.filter((entry) => !entry.achievement.NotInSteam).length;
+}
+
+export function AchievementCategory({
+  title,
+  achievements,
+  allAchievements,
+  sx,
+  children,
+}: React.PropsWithChildren<IProps>): JSX.Element {
+  // Most parts of the four categories in the old code were very similar (besides the content of
```

Comment:
These lines can probably be removed. Comments should document the code as it is, not in reference to as it was. It was useful to read right now, though.